lib/aliases.zsh

```diff
@@ -1,35 +0,0 @@
-# Push and pop directories on directory stack
-alias pu='pushd'
-alias po='popd'
-
-# Basic directory operations
-alias ...='cd ../..'
-alias -- -='cd -'
```

Comment:
Is there any special reason for removing `alias -- -='cd -'` while not adding it back to `directories.zsh
`?
(I got used to enter a single `-` instead of `cd -`, and it's no longer there now.)